src/www/ui_tests/api/Controllers/UploadTreeControllerTest.php

```diff
@@ -503,5 +544,64 @@ public function testGetHighlightEntries()
       $this->assertEquals($expectedResponse->getStatusCode(), $actualResponse->getStatusCode());
       $this->assertEquals($this->getResponseJson($expectedResponse), $this->getResponseJson($actualResponse));
     }
+
+    /**
+     * @test
+     * -# Test for UploadTreeController::getLicenseDecisions()
+     * -# Check if response status is 200 and RES body matches
+     * @throws \Fossology\Lib\Exception
+     */
+    public function testGetLicenseDecisions()
+    {
+      $itemId = 200;
+      $uploadId = 1;
+      $licenseId = 123;
+      $licenses = array();
+
+      $itemTreeBounds = new ItemTreeBounds($itemId, 'uploadtree_a', $uploadId, 0, 2);
```

Comment:
Having more than 1 difference between left and right makes an item tree contain a file. This causes test cases to fail.
```suggestion
      $itemTreeBounds = new ItemTreeBounds($itemId, 'uploadtree_a', $uploadId, 0, 1);
```

Reply:
Thank you @GMishx